client/test/components/conference_room/provider/EventProvider.spec.jsx

```diff
@@ -106,4 +111,82 @@ describe('<EventProvider />', () => {
       });
     });
   });
+
+  describe('confirming event', () => {
+    const wrapper = mount(<EventProvider conferenceRoom={conferenceRoom} component={DummyComponent} />);
+    const componentWrapper = wrapper.find(DummyComponent);
+
+    before(() => {
+      sinon.stub(EventSource, 'confirm');
+    });
+
+    after(() => {
+      EventSource.confirm.restore();
+    });
+
+    context('with successful request', () => {
+      before(() => {
+        wrapper.setState({ currentEvent: Event.build({ confirmed: false }) });
+        EventSource.confirm.resolves();
+        componentWrapper.find('button#confirm').simulate('click');
```

Comment:
you can move `componentWrapper.find('button#confirm')` to a separate variable. This lets you stay DRY